tests/Fixtures/Integration/priority/fully_qualified_strict_types,no_superfluous_phpdoc_tags.test

```diff
@@ -6,15 +6,17 @@ Integration of fixers: fully_qualified_strict_types,no_superfluous_phpdoc_tags.
 <?php
 // No namespace
 /**
+ * @param Foo  $foo
```

Comment:
needs https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7189 to be removed correctly